swig/python/CMakeLists.txt

```diff
@@ -262,13 +269,23 @@ if (Python_Interpreter_FOUND)
   if (DEFINED GDAL_PYTHON_INSTALL_PREFIX)
     file(TO_NATIVE_PATH "${GDAL_PYTHON_INSTALL_PREFIX}" NATIVE_GDAL_PYTHON_INSTALL_PREFIX)
     if (WIN32)
-      string(REPLACE "\\" "\\\\" NATIVE_GDAL_PYTHON_INSTALL_PREFIX "${NATIVE_GDAL_PYTHON_INSTALL_PREFIX}")
+	  if(NOT MINGW)
```

Comment:
lack of indentation

Reply:
See above comments. This piece of code should be maintained.